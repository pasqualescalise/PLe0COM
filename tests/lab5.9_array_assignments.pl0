VAR int_array[10] : int;
VAR short_array[10] : short;
VAR ubyte_array[10] : ubyte;
VAR boolean_array[10] : boolean;

BEGIN
	int_array = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]int;
	short_array = [0, 100, 200, 300, 400, 500, 600, 700, 800, 900]short;
	ubyte_array = [0, 100, 200, 300, 400, 500, 600, 700, 800, 900]ubyte;
	boolean_array = [True and True, False or False, True, False, True, False, True, False, True, False]boolean;

	print int_array;
	print short_array;
	print ubyte_array;
	print boolean_array;
	print [0, 1, 2, 3, 4]int;
	print [odd 0, odd 1, odd 2, odd 3, odd 4]boolean
END
