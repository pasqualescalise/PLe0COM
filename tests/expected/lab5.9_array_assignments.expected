[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
[0, 100, 200, 300, 400, 500, 600, 700, 800, 900]
[0, 100, 200, 44, 144, 244, 88, 188, 32, 132]
[True, False, True, False, True, False, True, False, True, False]
[0, 1, 2, 3, 4]
[False, True, False, True, False]
